### PR TITLE
Re-activate celestehorgan as approver for SIG Docs

### PR DESCRIPTION
**Pull request by celestehorgan**
This PR reactivates Celeste (Me!) as an approver, because I'm back baybeeee!!!!

### CODE REVIEW

**Comment by natalisucks** [comment]
hi @celestehorgan, welcome back to the reviewing team! i'd actually also recommend that you add yourself to the reviewers list, so that you can be in the pool of automatically assigned reviewers for contributions coming into docs

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -27,6 +27,7 @@ aliases:
     - rlenferink
   sig-docs-en-owners: # Admins for English content
     - divya-mohan0209
+    - celestehorgan
```
Comment (dipesh-rawat) [inline OWNERS_ALIASES:30]: _Very tiny nit:_ Would it be feasible to relocate your name to the line above for maintaining alphabetical order? Although I see there is one existing entry `(drewhagen)` from 1.30 Release Team inserted out of sequence below, maintaining alphabetical order in this file would be nice.

Also, **Welcome Back** 🎉

Comment (celestehorgan) [reply OWNERS_ALIASES:29]: Resolved in latest force push.

**Comment by celestehorgan** [comment]
> hi @celestehorgan, welcome back to the reviewing team! i'd actually also recommend that you add yourself to the reviewers list, so that you can be in the pool of automatically assigned reviewers for contributions coming into docs

> resolved in latest push

**Pull request merged**